packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -37,10 +37,16 @@ class CreateCommand extends FlutterCommand {
       help: 'Also add a flutter_driver dependency and generate a sample \'flutter drive\' test.'
     );
     argParser.addFlag(
-      'plugin',
+      'package',
```

Comment:
We can do it in this patch and just provide a legacy --plugin option to not break people in the meantime.

The thing to avoid is adding something we know we're going to change.